apps/meteor/tests/end-to-end/api/teams.ts

```diff
@@ -250,6 +257,51 @@ describe('[Teams]', () => {
 				.expect('Content-Type', 'application/json')
 				.expect(400);
 		});
+
+		it('should delete a team if an error occurs when creating a room for it', async () => {
```

Comment:
is this test still valid? Cause i don't see any deletion going on now 👀

Reply:
I think the test is still valid, but the description is outdated...